Specs/DateTime/Dutch/TimePeriodParser.json

```diff
@@ -112,14 +108,13 @@
     "Context": {
       "ReferenceDateTime": "2016-11-07T16:12:00"
     },
-    "NotSupported": "dotnet",
     "NotSupportedByDesign": "javascript,python,java",
     "Results": [
       {
         "Text": "tussen 17:00 en 18:00",
         "Type": "timerange",
         "Value": {
-          "Timex": "(T17,T18,PT1H)",
+          "Timex": "(T17:00,T18:00,PT1H)",
```

Comment:
Can you please confirm this is the same behaviour as in English for a similar input?

Reply:
confirm.
In English, the input "between 17:00 and 18:00" is parsed as 
(T17:00,T18:00,PT1H)
as the Dutch "tussen 17:00 en 18:00".
Instead, the input "5 to 6pm" is parsed as
(T17,T18,PT1H)
as the Dutch "tussen 5 en 6 's avonds"